includes/definitions/supermicro-bmc.yaml

```diff
@@ -0,0 +1,31 @@
+os: supermicro-bmc
+text: 'Supermicro-BMC'
+type: appliance
+icon: supermicro
+snmp_bulk: false
+mib_dir: supermicro
+group: supermicro
+over:
+    - { graph: device_temperature, text: 'Temperatures' }
+    - { graph: device_fanspeed, text: 'Fan Speed' }
+poller_modules:
+    applications: false
+    bgp-peers: false
+    mpls: false
+    ospf: false
+    stp: false
+    wireless: false
+discovery_modules:
```

Comment:
Is it necessary to explicitely disable these ? Does it slow down or fail in the discovery/polling if you keep them ?

Reply:
> Is it necessary to explicitely disable these ? Does it slow down or fail in the discovery/polling if you keep them ?

Since i have 1000+ device distributed on 5 poller i like to disable everything what take precious time
these modules will never be used on BMC. Do you agree ???
but, if you wish, i could remove these lines

Reply:
> So yes, please do not disable anything unless it actually crashes discovery or polling.

ok, removed from YAML